homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+4.
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White.
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language.
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website look pretty.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+(1) Source code storage and control
+(2) Collaboration
+(3) Tracking
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+When you fork a repo you create a instance of the repository in your own github account where you can make changes however you like.
 
 ## Question 10 
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+When you clone, you're actually downloading an entire copy of a repo from a website like Github to your local machine("personal computer").
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Pull Request is simply an event where a contributor ask's the maintainer of a Git repo to review code and grab(pull) the changes made.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see into the past of your repo.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+You send the changes of your local repository to Github.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A place for us to store and share our code. And collaborate with others.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+The "commit" command is used to save your changes to the local(personal computer) repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Code Studio
```

Comment:
Hi Selchuk, these are great answers. You just need a minor change in this answer . It is 'Visual Studio Code' .

Reply:
Hi Hatice, thank you for taking the time to review my work. However, it seems like you've misread the document.  The line highlighted in red is what's removed and the line highlighted in green is what's been added by me.  Please check the "View rendered homework.md" file at the top of the page for my answers.

Reply:
I agree with you. Only you can swap the words "Code" and  "Studio" to make your answer "Visual Studio Code". You can even ignore it as it is a minor thing :)

Reply:
> I agree with you. Only you can swap the words "Code" and "Studio" to make your answer "Visual Studio Code". You can even ignore it as it is a minor thing :)

Thank you for pointing this out.  I wouldn't have seen it in a million years otherwise.

Reply:
> Hi Selchuk, these are great answers. You just need a minor change in this answer . It is 'Visual Studio Code' .

I've just realized that I've misunderstood you, I'm sorry.

Reply:
No worries. It sometimes happens to me:)